scripts/prepare_sector_network.py

```diff
@@ -51,6 +51,33 @@ def add_lifetime_wind_solar(n):
         n.generators.loc[[index for index in n.generators.index.to_list()
                         if carrier in index], 'lifetime']=costs.at[carrier_name,'lifetime']
 
+
+def create_network_topology(n, prefix):
```

Comment:
Could work with defaults:
```suggestion
def create_network_topology(n, prefix="", connector = " -> "):
```

Reply:
Yes, I agree!